From 4084936ce294cd224ee270a99d50f97726c9008d Mon Sep 17 00:00:00 2001
From: Hien Huynh <hien.huynh.px@renesas.com>
Date: Tue, 3 Mar 2020 14:53:08 +0700
Subject: [PATCH 089/268] arm64: dts: renesas: hihope-common: enable msiof0
 node

This patch enable MSIOF0 on SoC R8A774A1 for HiHope Board.

Signed-off-by: Huynh Thanh Hung <hung.huynh.wz@renesas.com>
Signed-off-by: Hien Huynh <hien.huynh.px@renesas.com>
---
 arch/arm64/boot/dts/renesas/hihope-common.dtsi | 17 +++++++++++++++++
 1 file changed, 17 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/hihope-common.dtsi b/arch/arm64/boot/dts/renesas/hihope-common.dtsi
index f98958f..5100c22 100644
--- a/arch/arm64/boot/dts/renesas/hihope-common.dtsi
+++ b/arch/arm64/boot/dts/renesas/hihope-common.dtsi
@@ -13,6 +13,10 @@
 		serial1 = &hscif0;
 		serial2 = &hscif1;
 		serial3 = &scif0;
+		spi0 = &msiof0;
+		spi1 = &msiof1;
+		spi2 = &msiof2;
+		spi3 = &msiof3;
 	};
 
 	chosen {
@@ -347,6 +351,12 @@
 		groups = "usb30";
 		function = "usb30";
 	};
+
+	msiof0_pins: spi0 {
+		groups = "msiof0_clk", "msiof0_sync",
+			 "msiof0_rxd", "msiof0_txd";
+		function = "msiof0";
+	};
 };
 
 &rcar_sound {
@@ -516,3 +526,10 @@
 &vspi0 {
 	status = "okay";
 };
+
+&msiof0 {
+	pinctrl-0 = <&msiof0_pins>;
+	pinctrl-names = "default";
+
+	status = "okay";
+};
-- 
2.7.4

